src/main/java/com/checkout/payments/links/PaymentLinkResponse.java

```diff
@@ -1,13 +1,19 @@
 package com.checkout.payments.links;
 
 import com.checkout.common.Resource;
+import com.google.gson.annotations.SerializedName;
 import lombok.Data;
 import lombok.EqualsAndHashCode;
 
 @Data
 @EqualsAndHashCode(callSuper = true)
 public final class PaymentLinkResponse extends Resource {
+
     private String id;
+
+    @SerializedName("expires_on")
     private String expiresOn;
+
     private String reference;
+
```

Comment:
The attribute `created_on` is missing right?

https://github.com/checkout/checkout-api-reference/commit/f4bf0861669644ef037bce7b5115fa399b1bb1ec

Reply:
I don't think so.

This [file](https://github.com/checkout/checkout-api-reference/blob/7ce49e81b49e20710717d8a624b64054de2ba219/abc_spec/components/schemas/PaymentLinks/GetPaymentLinkResponse.yaml) belongs to PaymentLinkDetailsResponse

This [file](https://github.com/checkout/checkout-api-reference/blob/7ce49e81b49e20710717d8a624b64054de2ba219/abc_spec/components/schemas/PaymentLinks/PaymentLinksResponse.yaml) belongs to PaymentLinkResponse